docker-compose.development.yml

```diff
@@ -38,7 +38,8 @@ services:
       - "4200:4200"
     volumes:
       - "./web/src:/usr/src/app/src"
-    command: "ng build --watch"
+      - "./web/npm-cache:/root/.npm"
```

Comment:
Do we want / need to map this volume to a host directory? I could see the benefit perhaps being that you can share one `node_modules` between a locally running process and the dockerized version, but I'm not sure if that would actually work if the host OS was different than the container OS